File: Iniciante/1014 - Consumo/Python/1014.py
def main():
    total_distance_traveled = int(input())
    total_fuel_used = float(input())
    average_consumption = calculate_average_consumption(
        total_distance_traveled, total_fuel_used
    )
    print(f"{average_consumption:.3f} km/l")


def calculate_average_consumption(total_km, total_fuel):
    return total_km / total_fuel


if __name__ == "__main__":
    main()

File: Iniciante/1014 - Consumo/JavaScript/1014.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split("\n");
  const totalDistanceTraveled = parseInt(lines.shift());
  const totalFuelUsed = parseFloat(lines.shift());
  const averageConsumption = calculateAverageConsumption(totalDistanceTraveled, totalFuelUsed);
  console.log(`${averageConsumption.toFixed(3)} km/l`);
}

function calculateAverageConsumption(totalKm, totalFuel) {
  return totalKm / totalFuel;
}

main();